src/App.js

```diff
@@ -1,19 +1,35 @@
-import './App.css';
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/extensions */
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] As required by MIcroverse, you are not meant to use any LInters diable rule in your codebase. The Linters are there to make sure you follow the best practices. I'll suggest you remove these and try to fix the errors. :smile: :small_airplane: :+1: